test/test_unary_ufuncs.py

```diff
@@ -240,19 +284,17 @@ def assertEqualHelper(self, actual, expected, msg, *, dtype, exact_dtype=True, *
 
     # Tests that the function and its (array-accepting) reference produce the same
     #   values on a range of tensors, including empty tensors, scalar tensors,
-    #   1D tensors and a large 2D tensor with interesting and extremal values
-    #   and discontiguities.
+    #   1D tensors and a large 2D tensor with interesting and extremal values.
     @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
     @suppress_warnings
     @ops(unary_ufuncs)
     def test_reference_numerics(self, device, dtype, op):
-        include_extremals = (op.handles_complex_extremals if
-                             dtype in (torch.cfloat, torch.cdouble) else op.handles_extremals)
+        if op.ref is None:
+            raise unittest.SkipTest('Skipped! Op has no available reference.')
 
         tensors = generate_numeric_tensors(device, dtype,
                                            domain=op.domain,
-                                           include_large_values=op.handles_large_floats,
-                                           include_extremal_values=include_extremals)
+                                           include_large_values=op.handles_large_floats)
```

Comment:
is it better to set `include_large_values` arg for complex here? It's confusing that `generate_numeric_tensors` just ignores this argument for complex.

Reply:
Let's just rename the argument to "include_large_floats".